api/pom.xml

```diff
@@ -23,7 +23,7 @@
   <parent>
     <groupId>org.phenotips</groupId>
     <artifactId>clinical-text-analysis-extension</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>1.3-SNAPSHOT</version>
```

Comment:
This isn't right. This is a separate module, so it doesn't use the same versioning as the main project.